include/swift/Reflection/ReflectionContext.h

```diff
@@ -922,6 +923,47 @@ class ReflectionContext
     return llvm::None;
   }
 
+  llvm::Optional<std::string> iterateMetadataAllocationBacktraces(
+    std::function<void (StoredPointer, uint32_t, const StoredPointer *)> Call) {
+    std::string BacktraceListName =
```

Comment:
Oh, if we need a `std::string` there, then leave it be.  Otherwise, yes, it would ensure that it gets put into `.rdata` and avoids a constructor call.

Reply:
I checked a small test program and clang is able to optimize away the `std::string` parts entirely. Hopefully the same is true here.

Reply:
SGTM